src/HealthGPS.Console/command_options.cpp

```diff
@@ -11,10 +11,13 @@ cxxopts::Options create_options() {
     cxxopts::Options options("HealthGPS.Console", "Health-GPS microsimulation for policy options.");
     options.add_options()("f,file", "Configuration file full name.", cxxopts::value<std::string>())(
         "s,storage", "Path to root folder of the data storage.", cxxopts::value<std::string>())(
-        "j,jobid", "The batch execution job identifier.",
-        cxxopts::value<int>())("verbose", "Print more information about progress",
-                               cxxopts::value<bool>()->default_value("false"))(
-        "help", "Help about this application.")("version", "Print the application version number.");
+        "j,jobid", "The batch execution job identifier.", cxxopts::value<int>())(
+        "T,threads", "The maximum number of threads to create (0: no limit, default).",
```

Comment:
Why uppercase here?

Reply:
It seems to be standard for a lot of tools (e.g. `xz`) but I'm not sure why! It can be lowercase if you prefer.